### PR TITLE
[17.0][MIG] marketing_crm_partner: Migration to 17.0

### PR DESCRIPTION
**Pull request by MohamedOsman7**

### CODE REVIEW

**Comment by MohamedOsman7** [comment]
@flotho Could you please make a review?

**Review by CRogos — APPROVED** [review]
Code + Test LGTM

**Comment by MohamedOsman7** [comment]
@peluko00 @ppyczko could you please also here make a review

File: marketing_crm_partner/README.rst [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+5.  **Save**.
+6.  Click **Convert to Opportunity**.
+7.  Choose the option **Customer > Create a new customer**. |choose
+    customer|
+8.  Click **Create Opportunity**.
+9.  Click on the name of the newly linked partner. |new linked partner|
+10. Go to tab **Sales & Purchases**.
+11. There you have the new fulfilled fields in the marketing section.
+    |partner marketing|
+
```
Comment (peluko00) [inline marketing_crm_partner/README.rst]: The images don't load correctly

Comment (CRogos) [reply marketing_crm_partner/README.rst]: I think as soon as the PR is merged, the images should be visible. The images point to the 17.0 repo, which should be correct in my opinion.

**Review by peluko00 — APPROVED** [review]
LGTM, code review and tested in runboat

**Review by ppyczko — APPROVED** [review]
LGTM 👍

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
There are new images in SVG that shouldn't be in that format, but PNG, and corresponding to enterprise snapshots. They should be in community version.

**Comment by MohamedOsman7** [comment]
@pedrobaeza Thanks for the hint. I made the requested changes.

**Review by pedrobaeza — APPROVED** [review]
Thank you for the changes

/ocabot migration marketing_crm_partner
/ocabot merge nobump

**Pull request merged**